Dockerfile

```diff
@@ -100,6 +101,15 @@ RUN curl -sSL https://ftp.postgresql.org/pub/source/v$PQ_VER/postgresql-$PQ_VER.
     rm $PREFIX/lib/*.so && rm $PREFIX/lib/*.so.* && rm $PREFIX/lib/postgres* -rf &&  \
     cd .. && rm -rf postgresql-$PQ_VER
 
+# Build libsqlite3 using same configuration as the alpine linux main/sqlite package
```

Comment:
straight out of the https://git.alpinelinux.org/cgit/aports/tree/main/sqlite/APKBUILD?h=3.7-stable i presume?

Reply:
Yes.

Reply:
However I did not include the rpath removal because we are only using the static library portion of the build.